fontforge/http.c

```diff
@@ -663,7 +663,7 @@ return( true );
 FILE *URLToTempFile(char *url,void *_lock) {
     FILE *ret;
 
-    if ( strncasecmp(url,"http://",7)==0 )
+    if ( strncasecmp(url,"http://",7)==0 || strncasecmp(url,"https://",8)==0 )
```

Comment:
I'd rather not even have the ability for FontForge to open files directly from the internet. FF's implementation is horribly insecure/naive at best.